Makefile

```diff
@@ -1,22 +1,19 @@
-#Set language versions
-PY_IMG_TAG=3.7.9-stretch
-NODE_VER=lts
+# Set make variables from .env file
+ifneq (,$(wildcard ./.env))
+    include .env
+    export
+	ENV_FILE_PARAM = --env-file .env
+endif
 
-DOCKER_REGISTRY ?= docker.chameleoncloud.org
 DOCKER_TAG ?= $(shell git rev-parse --short HEAD)
-DOCKER_IMAGE := $(DOCKER_REGISTRY)/portal:$(DOCKER_TAG)
-DOCKER_IMAGE_LATEST := $(DOCKER_REGISTRY)/portal:latest
-
-PORTAL_MANAGE_CMD := docker-compose exec portal python manage.py
-#if APP is unset, then the variable equals the empty string.
-ifeq ($(APP),)
-MIGRATIONS_CMD := $(PORTAL_MANAGE_CMD) makemigrations
-else
-MIGRATIONS_CMD := $(PORTAL_MANAGE_CMD) makemigrations "$(APP)"
-endif
+DOCKER_IMAGE ?= $(DOCKER_REGISTRY)/portal:$(DOCKER_TAG)
+DOCKER_IMAGE_LATEST ?= $(DOCKER_REGISTRY)/portal:latest
+
+.env:
```

Comment:
I was going to suggest this use `.env.sample` as a dependency, then realized that would mean .env would get overridden when the sample changed. Yes, good to keep without dependencies then :)